configs/raspberrypi/uboot/include/raspberrypi5_64_config

```diff
@@ -1,4 +1,4 @@
-MENDER_DEVICE_TYPE="raspberrypi5"
+MENDER_DEVICE_TYPE="raspberrypi5_64"
```

Comment:
:+1: 

But please add a changelog entry to the rename because, in theory, someone could have been using raspberrypi5 integration from the previous mender-convert release (yeah, assuming the user found a workaround for u-boot or something, but still...)